### DISCUSSION TITLE
Why my project can't composite styled-components?

### DISCUSSION THREAD

**Discussion by coo-kiee** [styled-components/styled-components#4171] [Q&A] [answered]
I want to composite component with fetching function;

but, 'CustomStyledButton' don't have css itself.
such as, 
```
width: 100px;
background-color: #000;
```

https://codesandbox.io/embed/objective-shape-7r2pkw?fontsize=14&hidenavigation=1&theme=dark

**Comment by coo-kiee** [comment]
Oh... I resolved it
I don't add buttonAttributes to DefaultButton :b

**Comment by quantizor** [comment, marked as answer]
Resolved by the author — the issue was that attributes were not being passed to the base component.
